controllers/controllers/cluster_controller.go

```diff
@@ -116,5 +126,30 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *anywherev1.C
 }
 
 func (r *ClusterReconciler) reconcileDelete(ctx context.Context, cluster *anywherev1.Cluster, log logr.Logger) (ctrl.Result, error) {
+	capiCluster := &clusterv1.Cluster{}
+	capiClusterName := types.NamespacedName{Namespace: "eksa-system", Name: cluster.Name}
+	r.log.Info("Deleting", "name", cluster.Name)
+	err := r.client.Get(ctx, capiClusterName, capiCluster)
+
+	switch {
+	case err == nil:
+		r.log.Info("Deleting CAPI cluster", "name", capiCluster.Name)
+		if err := r.client.Delete(ctx, capiCluster); err != nil {
+			r.log.Info("Error deleting CAPI cluster", "name", capiCluster.Name)
```

Comment:
Is this necessary? We return the error, does that get logged at a higher level somewhere? Generally we should _either_ log (handle) the error, or return it.

Reply:
yes, for now. Future PRs will either change this type of calls to log.V(level).Info() or remove them.